__tests__/registries/npm-registry.js

```diff
@@ -175,6 +175,24 @@ describe('request', () => {
 
     expect(requestParams.headers.authorization).toBe('Bearer testAuthToken');
   });
+
+  test('should add authorization header with correct token for scoped package', () => {
```

Comment:
I don't believe it is.
This tests that a scope configured with a non-default registry is receiving the token specified for that scope.
The previous test is testing the use of the default _authToken when fetching from a scope that has no configured registry.
I'm happy to remove the previous test if you believe it does not add value.